spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+RSpec.describe PostsController, type: :controller do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get :index
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'GET #show' do
```

Comment:
[OPTIONAL] I suggest you to give followup guideline in your readme to how to test your app. Thank you

> EXTRA: it would be great idea if you could implement some prelaod data in your [seeds](https://medium.com/@SJTGs/seeding-in-rails-ff6b4e35941b).rb file to make your data loaded for testing.